src/database/prisma/schema.prisma

```diff
@@ -507,6 +507,9 @@ model Member {
     @@map("members")
 }
 
+/// @@allow('create', has(auth().policies, 'core:member:ping'))
```

Comment:
Does this mean these rows have to be updated whenever a new way to send notficiations is added? Perhaps we should just set "create" to "_" or whatever "logged in" is?